paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -163,6 +163,14 @@ class PaparazziPlugin : Plugin<Project> {
         test.outputs.dir(snapshotOutputDir)
 
         val paparazziProperties = project.properties.filterKeys { it.startsWith("app.cash.paparazzi") }
+        test.systemProperties.putAll(paparazziProperties)
+
+        // Explicitly register these as inputs so that they are considered when determining up-to-date.
+        // They have to be added later to test.systemProperties,
+        // because at this point in the configuration, the properties are not resolvable yet.
+        // They become resolvable after the last afterEvaluate runs.
```

Comment:
```suggestion
        // Explicitly register these as inputs so that they are considered when determining up-to-date.
        // The properties become resolvable after the last afterEvaluate runs.
```